mmdeploy/backend/ncnn/onnx2ncnn.py

```diff
@@ -32,26 +35,34 @@ def get_output_model_file(onnx_path: str, work_dir: str) -> List[str]:
     return [save_param, save_bin]
 
 
-def onnx2ncnn(onnx_path: str, save_param: str, save_bin: str):
+def from_onnx(onnx_model: Union[onnx.ModelProto, str],
+              output_file_prefix: str):
     """Convert ONNX to ncnn.
 
     The inputs of ncnn include a model file and a weight file. We need to use
     a executable program to convert the `.onnx` file to a `.param` file and
     a `.bin` file. The output files will save to work_dir.
 
     Example:
-        >>> from mmdeploy.backend.ncnn.onnx2ncnn import onnx2ncnn
+        >>> from mmdeploy.apis.ncnn import from_onnx
```

Comment:
The example in docstring is supposed to be updated, as the arguments are changed.